tests/test_project/tests/base.py

```diff
@@ -46,6 +55,21 @@ def assert_password_validation_False(self, user_number, password_number):
             else:
                 raise e
 
+    def assert_password_validation_False_number_of_times(self, user_number, num_of_times):
+        """
+        Asserts password validation is False a 'num_of_times' for the test user
+        """
+        for i in range(num_of_times):
+            self.assert_password_validation_False(user_number=user_number, password_number=i)
+
+    def assert_password_validation_True_number_of_times(self, user_number, num_of_times):
+        """
+        Asserts password validation is True a 'num_of_times' for the test user
+        """
+        for i in range(num_of_times):
+            self.assert_password_validation_True(user_number=user_number, password_number=i)
+
+
```

Comment:
@spokerman12, these could be combined into a single method which takes an additional boolean (`True` or `False`) and retrieves the method using `gettattr()` and then invokes it. What do you say?